assets/src/edit-story/components/colorPicker/gradientLine.js

```diff
@@ -35,27 +35,22 @@ import useKeyDeleteStop from './useKeyDeleteStop';
 import useKeyFocus from './useKeyFocus';
 import usePointerAddStop from './usePointerAddStop';
 import usePointerMoveStop from './usePointerMoveStop';
-import { LINE_LENGTH, LINE_WIDTH } from './constants';
+import { GRADIENT_STOP_SIZE, LINE_LENGTH, LINE_WIDTH } from './constants';
 
 const LINE_FULL_LENGTH = LINE_LENGTH + LINE_WIDTH;
 
 const Line = styled.div.attrs(({ stops }) => ({
   tabIndex: -1,
   style: generatePatternStyles({
     type: 'linear',
-    // "Push" the ends of the gradient in, so it starts and
-    // ends at (LINE_WIDTH / 2) px
-    stops: stops.map(({ color, position }) => ({
-      color,
-      position: (position * LINE_LENGTH + LINE_WIDTH / 2) / LINE_FULL_LENGTH,
-    })),
```

Comment:
@barklund It seemed to me in testing that without this position adjustment, the gradient line looked more similar to the actual element, however, I might not see correctly -- what was the issue with original positions here?

Reply:
Yes, this was due to the first and last stop not being at the very ends of the displayed gradient line, but pushed in a bit. But now a stop at 0% is actually at 0px offset, so this is not needed anymore.